common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -974,6 +975,12 @@
   "feature.wynntils.mythicBoxScaler.scale.description": "How much more should mythic boxes should be scaled by?",
   "feature.wynntils.mythicBoxScaler.scale.name": "Mythic Box Scale",
   "feature.wynntils.mythicFound.description": "Do various things when a mythic item is found.",
+  "feature.wynntils.mythicFound.lootChestDryBoxes": "%s dry boxes",
```

Comment:
What about this then? Is it supposed to go into this PR?

Reply:
Hmm that probably could have been separated yes. I was trying to add a banner to this string but found it wasn’t localized so I localized it.

Reply:
Ok, I see, so the PR is intentionally a mix between different fixes.

Can I ask you to separate these two concerns? So there is one PR that just adds the chat banner, and one PR for localizing the messages, and other similar changes. I think the chat banner stuff is just 👍 but the other changes looked like it could do with some extra scrutinity. There were some regressions from StyledText back to Component, for instance, that I did not like. (In general, we're trying to move away from Component to StyledText).

Reply:
Okay. I’ll try to separate them later.